modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -65,6 +65,11 @@ public void testExportImportObjectDefinition() throws Exception {
 		testExportImport(
```

Comment:
I think we can remove this test case, as it is no longer the "base" test, it also removes one of the JSON